--- go-mode-20170726.555/go-mode.el	2018-02-27 10:31:05.911198156 +0100
+++ go-mode.el	                    2018-02-27 10:30:48.631429630 +0100
@@ -620,7 +620,7 @@
     (if (go-in-string-or-comment-p)
         (goto-char point)
       (setq indent (go-indentation-at-point))
-      (if (looking-at (concat go-label-regexp ":\\([[:space:]]*/.+\\)?$\\|case .+:\\|default:"))
+      (if (looking-at (concat go-label-regexp ":\\([[:space:]]*/.+\\)?$\\|case .+:\\|case default:"))
           (cl-decf indent tab-width))
       (setq shift-amt (- indent (current-column)))
       (if (zerop shift-amt)
